src/app/app.component.html

```diff
@@ -8,6 +8,14 @@
       >
     </div>
     <div class="links">
+      <a target="_blank" href="/factorio">
+        Factorio
+      </a>
+      <span class="separator">|</span>
+      <a target="_blank" href="/dsp">
```

Comment:
I would add `*ngIf` tags to these links so that you don't show a link to the calculator you're already on. You can actually use it with an else, so it should be like:

```html
<a *ngIf="(data$ | async).isDsp; else dspLink" ...>Factorio</a>
<ng-template #dspLink><a ...>Dyson Sphere Program</a></ng-template>
```

Reply:
Done, changed text to <name> Calculator, as now just seeing Factorio or Dyson Sphere Program is a little unclear as to what they are linking to